tools/mob_suite/mob_typer.xml

```diff
@@ -16,7 +16,7 @@
    --min_mob_ident '$adv_param.min_mob_ident'
    --min_ori_ident '$adv_param.min_ori_ident'
    --min_mpf_ident '$adv_param.min_mpf_ident'
-    --outdir '.';
+    --outdir 'outdir';
```

Comment:
The `ln` above should be single quoted and ; should be replaced by &&
Actually all double quotes should be replaced by a single quote.